<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentWriter</class>
 <widget class="QWidget" name="EquipmentWriter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>476</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="minimumSize">
      <size>
       <width>252</width>
       <height>0</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scopePage">
      <attribute name="title">
       <string>Telescope</string>
      </attribute>
      <layout class="QHBoxLayout" name="_3" stretch="1,0">
       <item>
        <widget class="KListWidget" name="scopeListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="scopeIdLabel">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="scopeIdLineEdit">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="scopeVendorLabel">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="scopeVendorLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="scopeModelLabel">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="scopeModelLineEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="scopeDriverLabel">
             <property name="text">
              <string>Driver:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="scopeDriverComboBox">
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="scopeTypeLabel">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="KComboBox" name="scopeTypeComboBox">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Refractor</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Newtonian</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Maksutov</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Schmidt-Cassegrain</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Kutter (Schiefspiegler)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Cassegrain</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="scopeApertureLabel">
             <property name="text">
              <string>Aperture:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="KDoubleNumInput" name="scopeApertureSpinBox">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="label">
                <string/>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="sliderEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="scopeApertureUnitLabel">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="scopeFocalLengthLabel">
             <property name="text">
              <string>Focal Length:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="KDoubleNumInput" name="scopeFocalLengthSpinBox">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="scopeFocalLengthUnitLabel">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="scopeLightGraspLabel">
             <property name="text">
              <string>Light Grasp</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="KDoubleNumInput" name="scopeLightGraspSpinBox">
               <property name="focusPolicy">
                <enum>Qt::TabFocus</enum>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="sliderEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="scopeLightGraspUnitLabel">
               <property name="text">
                <string>%</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="scopeGraspDefinedCheckBox">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="scopeErectedOrientationLabel">
             <property name="text">
              <string>Erected Orientation:</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QCheckBox" name="scopeErectedCheckBox"/>
           </item>
           <item row="11" column="0">
            <widget class="QLabel" name="scopeTruesidedLabel">
             <property name="text">
              <string>True Sided Orientation:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QCheckBox" name="scopeTruesidedCheckBox"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="scopeOrientationDefinedLabel">
             <property name="text">
              <string>Orientation Defined:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QCheckBox" name="scopeOrientationDefinedCheckBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addNewScopeButton">
             <property name="text">
              <string>Add New...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveScopeButton">
             <property name="text">
              <string>Save Telescope</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeScopeButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="eyepiecePage">
      <attribute name="title">
       <string>Eyepiece</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,0">
       <item>
        <widget class="KListWidget" name="eyepieceListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="eyepieceIdLabel">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="eyepieceIdLineEdit">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="eyepieceVendorLabel">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="eyepieceVendorLineEdit"/>
           </item>
           <item row="4" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="KDoubleNumInput" name="eyepieceFocalLengthSpinBox">
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="eyepieceFocalLengthUnitLabel">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="eyepieceFocalLengthLabel">
             <property name="text">
              <string>Focal Length:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="eyepieceModelLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="eyepieceModelLabel">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="eyepieceIsZoomLabel">
             <property name="text">
              <string>Zoom Eyepiece:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="eyepieceIsZoomCheckBox"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="eyepieceMaxFocalLengthLabel">
             <property name="text">
              <string>Maximal Focal Length:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="KDoubleNumInput" name="eyepieceMaxFocalLengthSpinBox">
               <property name="focusPolicy">
                <enum>Qt::TabFocus</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="eyepieceMaxFocalLengthUnitLabel">
               <property name="text">
                <string>mm</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="eyepieceFovLabel">
             <property name="text">
              <string>Apparent FOV:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KDoubleNumInput" name="eyepieceFovSpinBox">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="eyepieceFovUnitLabel">
             <property name="text">
              <string>Unit:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="eyepieceFovUnitComboBox">
             <item>
              <property name="text">
               <string>deg</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>rad</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arcsec</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>arcmin</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="eyepieceFovDefinedCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addNewEyepieceButton">
             <property name="text">
              <string>Add New...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveEyepieceButton">
             <property name="text">
              <string>Save Eyepiece</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeEyepieceButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lensPage">
      <attribute name="title">
       <string>Lens</string>
      </attribute>
      <layout class="QHBoxLayout" name="_4" stretch="1,0">
       <item>
        <widget class="KListWidget" name="lensListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout3">
         <item>
          <layout class="QFormLayout" name="formLayout3">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="lensIdLabel">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="lensIdLineEdit">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lensVendorLabel">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="lensVendorLineEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lensFactorLabel">
             <property name="toolTip">
              <string>Magnification factor of the Barlow/Shapley lens</string>
             </property>
             <property name="text">
              <string>Factor:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="KDoubleNumInput" name="lensFactorSpinBox">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="lensModelLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="lensModelLabel">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout3">
           <item>
            <spacer name="horizontalSpacer31">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addNewLensButton">
             <property name="text">
              <string>Add New...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveLensButton">
             <property name="text">
              <string>Save Lens</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeLensButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="filterPage">
      <attribute name="title">
       <string>Filter</string>
      </attribute>
      <layout class="QHBoxLayout" name="_5" stretch="1,0">
       <item>
        <widget class="KListWidget" name="filterListWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout4">
         <item>
          <layout class="QFormLayout" name="formLayout4">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::ExpandingFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="filterIdLabel">
             <property name="text">
              <string>Id:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="KLineEdit" name="filterIdLineEdit">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="filterVendorLabel">
             <property name="text">
              <string>Vendor:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="KLineEdit" name="filterVendorLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="filterModelLabel">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="KLineEdit" name="filterModelLineEdit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="filterColorLabel">
             <property name="text">
              <string>Color:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="filterTypeLabel">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="filterTypeComboBox"/>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="filterColorComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout7">
           <item>
            <spacer name="horizontalSpacer3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addNewFilterButton">
             <property name="text">
              <string>Add New...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="saveFilterButton">
             <property name="text">
              <string>Save Filter</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeFilterButton">
             <property name="text">
              <string>Remove</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="imagersPage">
      <attribute name="title">
       <string>Imagers</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <class>KDoubleNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>scopeListWidget</tabstop>
  <tabstop>scopeVendorLineEdit</tabstop>
  <tabstop>scopeModelLineEdit</tabstop>
  <tabstop>scopeDriverComboBox</tabstop>
  <tabstop>scopeTypeComboBox</tabstop>
  <tabstop>scopeApertureSpinBox</tabstop>
  <tabstop>scopeFocalLengthSpinBox</tabstop>
  <tabstop>addNewScopeButton</tabstop>
  <tabstop>saveScopeButton</tabstop>
  <tabstop>removeScopeButton</tabstop>
  <tabstop>eyepieceListWidget</tabstop>
  <tabstop>eyepieceVendorLineEdit</tabstop>
  <tabstop>eyepieceModelLineEdit</tabstop>
  <tabstop>eyepieceFocalLengthSpinBox</tabstop>
  <tabstop>eyepieceFovSpinBox</tabstop>
  <tabstop>eyepieceFovUnitComboBox</tabstop>
  <tabstop>addNewEyepieceButton</tabstop>
  <tabstop>saveEyepieceButton</tabstop>
  <tabstop>removeEyepieceButton</tabstop>
  <tabstop>lensListWidget</tabstop>
  <tabstop>lensVendorLineEdit</tabstop>
  <tabstop>lensModelLineEdit</tabstop>
  <tabstop>lensFactorSpinBox</tabstop>
  <tabstop>addNewLensButton</tabstop>
  <tabstop>saveLensButton</tabstop>
  <tabstop>removeLensButton</tabstop>
  <tabstop>filterListWidget</tabstop>
  <tabstop>filterVendorLineEdit</tabstop>
  <tabstop>filterModelLineEdit</tabstop>
  <tabstop>addNewFilterButton</tabstop>
  <tabstop>saveFilterButton</tabstop>
  <tabstop>removeFilterButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
